Volunteer-Project-Management.md

```diff
@@ -0,0 +1,57 @@
+### Agile Project Management Terminology
+
+During the Free Code Camp volunteer projects we use an Agile Project Management methodology. For many of our campers and stakeholders, this is their first exposure to Agile. While you're not expected to be an Agile expert, here are a few terms it would be helpful to know:
+
+
++ **Task Board** - A chart that lists what needs to be done, what's being worked on, and what's been completed. We use Trello as our task board.
++ **Sprint** - A period of development. We generally use a one week sprint length.
++ **Sprint Conference** - A meeting held at the end of each sprint to discuss the progress made and solicit feedback. Also commonly referred to as a sprint review.
++ **Stakeholder** - Someone outside the development team who has an interest in the project and its development.
++ **User Story** - A requirement or feature that the product must meet/include.
+
+For additional information on Agile you can check out the [agile management](https://en.wikipedia.org/wiki/Agile_management) page on Wikipedia or take a look at the [Agile Project Management for Dummies Cheat Sheet](http://www.dummies.com/how-to/content/agile-project-management-for-dummies-cheat-sheet.html).
+___
+
+### Tools we use
+
+[GoToMeeting](http://www.gotomeeting.com) - We use GoToMeeting to hold our online project management conferences.
+
+[When2Meet](http://www.when2meet.com) - We use When2Meet to share our availability with the team.
+
+[Trello](http://www.trello.com) - We use Trello to track our volunteer web development projects.
+
+[Google Calendar](https://www.google.com/calendar) - We use Google Calendar to track our project conferences.
+___
+
+### Trello Board Standard Layout
+
+When setting up the Trello board for your project, create the following lists (in this order left to right).
+
++ **General Information** - This list will include general information that might be useful to the team such as links to current sites, a description of the current or proposed workflow, or any other information that could be useful to the development team.
++ **Project Scope** - This list provides a general outline of what the project should accomplish.
++ **User Stories** - These are the <em>specific</em> features that the development must include.
++ **In Progress** - These are the user stories that the team is currently working on.
++ **Recently Completed** - These are the user stories that the team has completed since the last project conference.
++ **Demo'd** - These are the user stories that have been demo'd to, and accepted by, the client.
+
+___
+
+### Organizing a Scope Call
+In Progress.
```

Comment:
Jason, what feedback are you looking for?

@Rafase282 ,
If there is no issue with the current PR can we commit and then we can work together ( Jason, the other PMs, and myself) to add additional information?